service/src/main/java/tds/config/web/endpoints/ConfigController.java

```diff
@@ -31,4 +33,17 @@ public ConfigController(ConfigService configService) {
 
         return ResponseEntity.ok(clientSystemFlag);
     }
+
+    @GetMapping(value = "{clientName}/messages/{context}/{messageKey}/{language}", produces = MediaType.APPLICATION_JSON_VALUE)
+    @ResponseBody
+    ResponseEntity<String> getClientSystemMessage(@PathVariable final String clientName,
+                                                  @PathVariable final String language,
+                                                  @PathVariable final String context,
+                                                  @PathVariable final String messageKey,
+                                                  @RequestParam(required = false) final String subject,
+                                                  @RequestParam(required = false) final String grade) {
```

Comment:
Not sure I love this long URL.  I'm up for changing it.

Also worth noting, that I'm not sure grade and subject are used in the TDS application currently, but figured we should move it across because it might need to be used in practice at some point.